src/module/dialog-flow.component.ts

```diff
@@ -25,7 +26,17 @@ export class DialogFlowService {
 			);
 		}
 
-		const fulfillment = await handler.call(this, dialogFlowResponse);
+		if (matchedHandlers.length > 1) {
+			throw new Error(`Unable to process multiple handlers [${matchedHandlers.map((handler, key) => key).join(', ')}]`);
```

Comment:
`Unable to process multiple handlers [${matchedHandlers.map(({provider, methodName}) => methodName).join(',/')} for intent[${intent}]/action[${action}]`